### PR TITLE
Show two waterfalls on example page.

### PR DESCRIPTION
**Pull request by tobli**
This is two make it easier to catch changes that break multiple waterfalls on the same page. Detected and corrected some minor issues with the example page css to make it work with more than one waterfall.

### CODE REVIEW

**Comment by tobli** [comment]
Not the prettiest code/page, but gets the job done.

File: src/index.html [hunk 1]
```diff
@@ -35,62 +39,60 @@
         /** holder DOM element to render PerfCascade into */
         var outputHolderEl = document.getElementById("output")
         var pageSelectorEl = document.getElementById("page-selector")
+        var outputHolder2El = document.getElementById("output2")
+        var pageSelector2El = document.getElementById("page-selector2")
         var legendHolderEl = document.getElementById("legendHolder")
 
-
+    function setup(options, fileinputId, outputHolder) {
```
Comment (micmro) [inline src/index.html]: looks like the indentation is off here

**Review by micmro — APPROVED** [review]
Could you fix the indentation? Besides looks good.

**Comment by tobli** [comment]
I fixed the indentation, amended, force pushed and will merge now.

**Pull request merged**